packages/vite/src/node/plugins/css.ts

```diff
@@ -897,6 +898,13 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         })
       }
 
+      const chunkImportMap = config.build.chunkImportMap
+        ? createChunkImportMap(bundle)
+        : {}
```

Comment:
Probably we need to think about how to let other plugins access this information. I assume other plugins will need to access this information sometimes.